diagnostic.md

```diff
@@ -35,7 +39,10 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    One difference is in the first block, product is nested inside of products.
+    The second difference is the path's are different, the second block acts
+    as the root route for product rather than products since this is the top
+    most route for product.
```

Comment:
You've almost got it.  These two route definitions are identical  URL paths.  The distinction is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The not-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.